infer/bin/inferTraceBugs

```diff
@@ -187,9 +187,7 @@ class Selector(object):
         if len(self) > 1:
             report_number_str = raw_input(
                 'Choose report to display (default=0): ')
-            if report_number_str == '':
-                report_number = 0
-            else:
+            if report_number_str != '':
                 report_number = self.parse_report_number(report_number_str)
         elif len(self) == 1:
             print('Auto-selecting the only report.')
```

Comment:
this branch also sets `report_number = 0` (on the line just below; can't click on it from github apparently...). Could you please remove that line as well and send a new pull request?